components/HomeHeader.vue

```diff
@@ -0,0 +1,192 @@
+              >
+              <v-btn
+                min-width="12em"
+                outlined
+                rounded
+                :to="`item/${item.Id}`"
+                >{{ $t('viewDetails') }}</v-btn
+              >
+            </v-col>
+          </v-row>
+        </v-container>
+      </div>
+    </swiper-slide>
+    <div slot="pagination" class="swiper-pagination"></div>
+  </swiper>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { SwiperOptions } from 'swiper';
+import { BaseItemDto, ImageType, ItemFields } from '~/api';
+import htmlHelper from '~/mixins/htmlHelper';
+import imageHelper from '~/mixins/imageHelper';
+
+export default Vue.extend({
+  mixins: [htmlHelper, imageHelper],
+  data() {
+    return {
+      items: [] as BaseItemDto[],
+      swiperOptions: {
+        autoplay: {
+          delay: 20000
+        },
+        initialSlide: 1, // For some reason, slide 0 is the last slide, when loop is enabled
+        loop: true,
+        effect: 'slide',
+        pagination: {
+          el: '.swiper-pagination'
+        }
+      } as SwiperOptions
+    };
+  },
+  async beforeMount() {
+    this.items = (
+      await this.$api.userLibrary.getLatestMedia({
+        userId: this.$auth.user.Id,
+        limit: 10,
+        fields: [ItemFields.Overview],
+        enableImageTypes: [ImageType.Backdrop, ImageType.Logo],
+        imageTypeLimit: 1
+      })
+    ).data;
+  },
+  methods: {
+    getBackdrop(item: BaseItemDto): string {
+      // TODO: Improve the image mixin and move this there
+      if (item.Type === 'Episode') {
+        return `${this.$axios.defaults.baseURL}/Items/${item.SeriesId}/Images/Backdrop`;
+      } else if (item.Type === 'MusicAlbum') {
+        return `${this.$axios.defaults.baseURL}/Items/${item?.AlbumArtists?.[0].Id}/Images/Backdrop`;
+      } else {
+        return `${this.$axios.defaults.baseURL}/Items/${item.Id}/Images/Backdrop`;
+      }
+    },
+    getLogo(item: BaseItemDto): string {
+      // TODO: Improve the image mixin and move this there
+      if (item.Type === 'Episode') {
+        return `${this.$axios.defaults.baseURL}/Items/${item.SeriesId}/Images/Logo`;
+      } else {
+        return `${this.$axios.defaults.baseURL}/Items/${item.Id}/Images/Logo`;
+      }
+    },
+    getOverview(item: BaseItemDto): string {
+      if (item.Overview) {
+        return this.sanitizeHtml(item.Overview);
+      } else {
+        return '';
+      }
+    }
+  }
+});
+</script>
+
+<style lang="scss" scoped>
+.swiper {
+  margin-bottom: -128px !important;
+}
+
+.slide-backdrop {
+  padding-bottom: 46.25%;
+  background-position: right center;
+  background-size: contain;
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+  mask-image: linear-gradient(
+      180deg,
+      rgba(18, 18, 18, 1) 60%,
+      rgba(18, 18, 18, 0) 100%
+    ),
+    linear-gradient(90deg, rgba(18, 18, 18, 1) 20%, rgba(18, 18, 18, 0) 70%);
+  mask-composite: subtract;
```

Comment:
```suggestion
  mask-composite: subtract;
  -webkit-mask-composite: source-out;
```

Should fix it.

See [here](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-mask-composite)

Reply:
Hm, we should have autoprefixer, which should do this automatically. I'll investigate.

Reply:
The auto prefixed is working but it is outputting '-webkit-mask-composite: subtract' which isn't a valid propert

Reply:
Oh, okay, I get it. Autoprefixer handles it, but it doesn't convert between options. Since subtract doesn't exist for the webkit version, it doesn't work.

Reply:
Fixed in 4ced6d0 :tada:

Reply:
Ah yeah that's right. 🎉